packages/dotcom-ui-flags/src/components/FlagsEmbed.ts

```diff
@@ -1,4 +1,5 @@
-import React from 'react'
+// TODO: Figure out why importing the package name does not work.
+import { ClientEmbed } from '../../../dotcom-ui-client-embed/src'
```

Comment:
I'm going to have another go at this but I could not figure out why referencing the package name does not work.

Reply:
I think this was because when I added these as a dependency I just had not ran `npm install` on the directory.

Reply:
I will pull your branch and have a play.

Reply:
Ah yes, that tripped me up too. If there are packages with dependencies on other packages then you need to install each of them (in order, I would imagine).